src/_includes/services_page_intro.liquid

```diff
@@ -1,5 +1,40 @@
 <header>
-  <div class="services-page-intro__image"></div>
+  <div class="services-page-intro__image">
+    <div class="services-page-intro__navigation">
+      <div class=services-page-intro__icon-container>
```

Comment:
I'm wondering whether we would need these container elements, or whether we could style and layout the a tags similarly. Not a major blocker for merging this one.